blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/PipelineImpl.java

```diff
@@ -179,4 +179,9 @@ public Resource resolve(Item context, Reachable parent, Item target) {
 
     }
 
+    @Exported(inline = true)
+    @Navigable
+    public Container<Resource> getActivities() {
+        return Containers.fromResource(getLink(),Lists.newArrayList(Iterators.concat(getQueue().iterator(), getRuns().iterator())));
```

Comment:
@michaelneale Yes I know. I would like to enforce default page size, but first let UI implement pagination or we won't have all pipelines appearing. So no I am not happy with how it is and waiting for UI to finish implementing pagination for all list items:)